spec/shared_examples/user_shared_examples.rb

```diff
@@ -488,6 +503,32 @@ def self.matches?(crypted, *tokens)
 
       expect(user.crypted_password).to eq Sorcery::CryptoProviders::SHA512.encrypt('secret', user.salt)
     end
+
+    context '' do
```

Comment:
Using an empty context feels like a bad practice. Can we update this to reflect why it's being used, or remove it?